c2corg_ui/static/js/feed.js

```diff
@@ -204,4 +235,31 @@ app.FeedController.prototype.documentType = function(type) {
   return app.utils.getDoctype(type).slice(0, -1);
 };
 
+/**
+ * Generate URL address of a document
+ * @param doc {Object}
+ * @returns {string}
+ */
+app.FeedController.prototype.getDocHref = function(doc) {
+  return '/' + app.utils.getDoctype(doc.document.type) + '/'
+    + doc.document.document_id + '/' + doc.lang + '/' + doc.document.title;
+};
+
+/**
+ * Generate URL address to the changed document version
+ * @param doc {Object}
+ * @returns {string | boolean}
```

Comment:
It doesn't make sense that a function returns that unconsistent outputs. Make a choice :P